tag/main.go

```diff
@@ -94,11 +100,26 @@ func getDockerImageTag() (string, error) {
 	if err != nil {
 		return "", errors.Wrap(err, "compile regex")
 	}
+	excludedRegexs, err := excludedTagRegexps()
+	if err != nil {
+		return "", errors.Wrap(err, "getting excluded regexes")
+	}
 	for _, tag := range tags {
 		if !regex.Match([]byte(tag)) {
 			fmt.Println("Skipping", tag)
 			continue
 		}
+		var exclude bool
+		for _, excludedRegex := range excludedRegexs {
+			if excludedRegex.Match([]byte(tag)) {
+				exclude = true
+				break
+			}
+		}
+		if exclude {
+			fmt.Println("Skipping", tag, "(explicitly excluded)")
```

Comment:
If we move this line up to where we set `exclude = true`, we can also print the regexp that caused it to be excluded, which might help if we end up debugging this later.

Reply:
good call, done!